lib/metrics.js

```diff
@@ -1,17 +1,31 @@
 import _ from 'underscore';
+import performance from 'react-native-performance';
 import MDTable from './MDTable';
 
+const decoratedAliases = new Set();
+
 /**
- * Each key is a method name and the value is an array of calls metadata
- * @type {Record<String, Array<Object>>}
+ * Capture a start mark to performance entries
+ * @param {string} alias
+ * @param {Array<*>} args
+ * @returns {{name: string, startTime:number, detail: {args: [], alias: string}}}
  */
-let stats = {};
+function addMark(alias, args) {
+    return performance.mark(alias, {detail: {args, alias}});
+}
 
-/* For some reason `performance.now()` does not start from `0` but a very large offset
-* like `508,080,000` see: https://github.com/facebook/react-native/issues/30069
-* Capturing an offset allows us to record start/ends times relative to app launch time
-* It also allows us to compare start/end times relative to the last resetMetrics */
-let PERFORMANCE_OFFSET = performance.now();
+/**
+ * Capture a measurement between the start mark and now
+ * @param {{name: string, startTime:number, detail: {args: []}}} startMark
+ * @param {*} detail
+ */
+function measureMarkToNow(startMark, detail) {
+    performance.measure(`${startMark.name} [${startMark.detail.args.toString()}]`, {
+        start: startMark.startTime,
+        end: performance.now(),
```

Comment:
Is using `performance.now()` the same as doing 

```js
performance.mark('endMark');
performance.measure('someMeasure', 'startMark', 'endMark');
```

Reply:
Yes, and originally I did it like that but then I noticed we're not really using the marks - only measures appear on the timeline, so we skip capturing a mark here
Do you want to capture an end mark first and then use marks to make the measure?

Reply:
Nah mostly just asking to see if these were equivalent - I didn't realize we could pass start and end to measure as options.